packages/cli/__tests__/zosfiles/__integration__/create/classicPds/__snapshots__/cli.files.create.dataset.classic.integration.test.ts.snap

```diff
@@ -192,13 +192,18 @@ exports[`Create Classic Data Set should display the help 1`] = `
 
       $ zowe zos-files create data-set-classic NEW.CLASSIC.DATASET
 
+   - Create an empty z/OS 'classic' PDSE using data set type
+   LIBRARY:
```

Comment:
funny how this one wraps around 😋

Reply:
Yes, I tried using double quotes around the string to match the other strings, but same result. I don't know what's causing it.

Reply:
This may be a bug, there are some issues in the Imperative repo about the example description being wrapped prematurely:
* https://github.com/zowe/imperative/issues/307
* https://github.com/zowe/imperative/issues/434